core/amber/src/main/python/core/models/marker.py

```diff
@@ -1,11 +1,44 @@
 from dataclasses import dataclass
-
+from pyarrow import Table
+from pandas import DataFrame
 
 @dataclass
 class Marker:
     pass
 
-
 @dataclass
 class EndOfUpstream(Marker):
     pass
+
+@dataclass
+class State(Marker):
```

Comment:
please add type annotations in function signatures.

Reply:
This is the class definition, and a type annotation shouldn't be required.

Reply:
Fixed.